samples/boards/up_squared/gpio_counter/README.rst

```diff
@@ -0,0 +1,70 @@
+..up_squared gpio_counter:
+
+UP Squared
+##########
+
+Overview
+********
+This sample provides an example of how to configure GPIO input and output to
+the UP Squared board.
+
+The sample enables a pin as GPIO input (active high) that triggers the increment
+of a counter (range is 0 to size of counter_pins[]). The counter increments for
+each change from 0 to 1 on HAT Pin 16. The value of the counter is represented
+on GPIO output (active high) as a 4-bit value
+(bin 0, 1, 2, 3 -> HAT Pin 35, 37, 38, 40).
+
+
++------------+-----+-----+-----+-----+-----+
+| Bit (bin)  | n/a |   3 |   2 |   1 |   0 |
++------------+-----+-----+-----+-----+-----+
+| HAT Pin    |  16 |  40 |  39 |  37 |  35 |
++------------+-----+-----+-----+-----+-----+
+| BIOS Pin   |  19 |  38 |  27 |  15 |  14 |
++------------+-----+-----+-----+-----+-----+
+| Direction  |  IN | OUT | OUT | OUT | OUT |
++------------+-----+-----+-----+-----+-----+
+| Active     |   H |   H |   H |   H |   H |
++------------+-----+-----+-----+-----+-----+
+
+
+Requirements
+************
+
+The application requires an UP Squared board connected to the PC through USB
+for serial console. The BIOS settings must be updated as specified in the
+source code comments for HAT Configurations (see table above).
+
+
+References
+**********
+
+- https://docs.zephyrproject.org/latest/boards/x86/up_squared/doc/index.html
+
+
+Building and Running
+********************
+
+Build the sample in the following way:
+.. zephyr-app-commands::
+    :zephyr-app: samples/boards/up_squared/gpio_counter
```

Comment:
Need a blank line before the directive (line 49) and previous sentence, and these parameters should be indented to the ``z`` in ``zephyr-app-commands``

Reply:
Thanks for clarifying!